fabtests/man/man7/fabtests.7

```diff
@@ -304,7 +304,7 @@ Tests memory registration.
 .RE
 .TP
 .B \f[I]fi_resource_freeing\f[]
-Allocates and closes fabric resources to check for proper cleanup.
+Allocates and closes fabric resources to check for proper cleanup. 
```

Comment:
Wrong patch. Still here.

Reply:
I'm going to go ahead and merge this series even though this change is wrong and removed in a separate patch.  But only because a daemon is going to blast over this man page anyway, as only md pages should ever be updated directly.